qxf2/qxf2-page-object-model
# Issue 348 remote env drishya

We have implemented the remote_credentials.py as env.remote 

Changed files : 

.circleci/config.yml
conf/browser_os_name_conf.py
conf/remote_credentials.py
page_objects/Base_Page.py
page_objects/driverfactory.py
page_objects/drivers/remote_options.py
utils/BrowserStack_Library.py
utils/interactive_mode.py

Added file 

env_remote   as replacement for remote_credentials_conf

Deleted file 
conf/remote_credentials.py


How to test : 

Switch to branch 
Rename the file env_remote as ".env.remote"
Provide the BS or SL details into it 
Execute /Run the pytest 
you could see the browserstack link with the test running on it.
